packages/electron/src/telemetry/electron-telemetry.ts

```diff
@@ -133,67 +175,73 @@ export const logSolutionMetrics = (response: any, time: number) => {
       });
 
       const SolutionMetrics = {
-        Metrics: { Status: "success" },
-        TimeStamp: new Date(),
-        ListMetrics: {},
-        Dimensions: [
-          { Name: "metricsType", Value: "Solutions" },
-          {
-            Name: "SolutionPath",
-            Value: crypto
-              .createHash("sha256")
-              .update(solutionDetails.solutionFilePath || "")
-              .digest("hex"),
-          },
-          { Name: "portingAssistantVersion", Value: app.getVersion() },
-          {
-            Name: "targetFramework",
-            Value: targetFramework,
-          },
-        ],
-      };
-
-      metricsBuffer.push(JSON.stringify(SolutionMetrics));
-      solutionDetails.projects.forEach((project) => {
-        const projectMetrics = {
-          Metrics: {
-            numNugets: {
-              Value: project.packageReferences?.length || 0,
-              Unit: "Count",
-            },
-            numReferences: {
-              Value: project.projectReferences?.length || 0,
-              Unit: "Count",
-            },
-          },
+        Type: PORTING_ASSISTANT_METRIC,
+        Content: {
```

Comment:
I think you could remove this metrics

Reply:
We don't need these metrics in local log files?

Reply:
All metrics should be moved to C# side